pages/index.tsx

```diff
@@ -283,14 +297,19 @@ const Index = () => {
                                 {tipo === "5" ?
                                     <Input type="text" id="chave" value={chaveAleatorio}
                                            onChange={(evt) => setChaveAleatorio(evt.target.value)}
-                                           placeholder="Chave Pix - Chave Aleatória" tabIndex={2} maxLength={70}
+                                           placeholder="Chave Pix - Chave Aleatória" tabIndex={2} maxLength={36}
                                            disabled={gerandoQrCode}/>
                                     : <></>}
                             </div>
                         </div>
                         <p className="text-sm text-slate-500">
                             <span
-                                className="pr-3 font-medium leading-none text-slate-700 dark:text-slate-300">Tamanho: {chave().length - (tipo === "3" || tipo === "4" ? 3 : 0)}/{chaveMaxSize()}</span>
+                                className="pr-3 font-medium leading-none text-slate-700 dark:text-slate-300">
+                                Tamanho: {chave().length - (tipo === "3" || tipo === "4" ? 3 : 0)}/{chaveMaxSize()}
+                            </span>
+                            <span className="text-red-700 font-semibold mr-1">
+                                {!isValidChave() ? `A chave digitada é inválida.` : ''}
+                            </span>
```

Comment:
What do you think of adding/removing the whole `<span>`?

```suggestion
                            {!isValidChave() ? 
                                <span className="text-red-700 font-semibold mr-1">
                                    A chave digitada é inválida.
                                </span>
                                :
                                <></> }
```